mir_common/mir_description/urdf/grippers/dynamixel/d435_center/dynamixel_gripper.urdf.xacro

```diff
@@ -20,6 +20,11 @@
       <child link="${name}_motor_mount_link" />
     </joint>
 
+    <joint name="${name}_static_grasp_link_joint" type="fixed" >
+        <origin xyz="0 0 0.08" rpy="0 0 0"/>  
```

Comment:
I think the z value (0.08) should be passed through a parameter because this is not the final value, needs to calibrated, and different gripper sizes will have different values. @RaghuvirShirodkar please have a look.

Reply:
I have made a variable for grasp link in https://github.com/b-it-bots/mas_industrial_robotics/pull/70/commits/cdac39a421c5352136d0c49c4b382756b5cb2f8b. It is possible to pass an argument to xacro file as mentioned [here](http://wiki.ros.org/xacro#Rospack_commands). However, we include this xacro file in [another xacro file](https://github.com/b-it-bots/mas_industrial_robotics/blob/kinetic/mir_robots/mir_hardware_config/youbot-brsu-4/urdf/robot.urdf.xacro). Now, as far as I could find on internet, it is not possible to provide argument to an included xacro file (quote and reference below)

> when including I don't think you can pass arguments like a roslaunch include, only the top level a.xacro can use xacro arg to get arguments passed from command line when expanded.

[ref](https://github.com/ros/xacro/issues/152#issuecomment-303489699)